lib/enum/basic_enum.rb

```diff
@@ -22,48 +22,58 @@ def number_of_elements_in_collection
   # This one also uses 'count'
   # Try it yourself!
   def number_of_floats_or_fixnums
-  end
+    collection.select {|num| num.is_a?(Numeric)}.count
+end
```

Comment:
Careful of your indentation here, it looks like this end is for the class MyEnumerables instead of the method.